go.mod

```diff
@@ -12,11 +12,12 @@ require (
 	github.com/sirupsen/logrus v1.8.1
 	github.com/songgao/water v0.0.0-20200317203138-2b4b6d7c09d8
 	github.com/stretchr/testify v1.7.1
-	github.com/urfave/cli/v2 v2.6.0
+	github.com/urfave/cli/v2 v2.7.1
 	golang.org/x/sys v0.0.0-20220319134239-a9b59b0215f8
 )
 
 require (
+	github.com/antzucaro/matchr v0.0.0-20210222213004-b04723ef80f0 // indirect
```

Comment:
This GPL2 dependency is incompatible with Apache License 2.0
https://github.com/urfave/cli/issues/1404

Reply:
Reverted in https://github.com/rootless-containers/rootlesskit/pull/306